### DISCUSSION TITLE
Add Change access option to the Custom Widget

### DISCUSSION THREAD

**Discussion by sandeep859** [Azure/api-management-developer-portal#2625] [Feature request]
We are creation a Custom widget on the managed portal as per this [link](https://learn.microsoft.com/en-us/azure/api-management/developer-portal-extend-custom-functionality).

We created a custom widget and we currently dont see an option to changes its access. Screen shot added here.
Is there a way to add the change access option to the custom widget. If not can we have a feature request for this.

![image](https://github.com/Azure/api-management-developer-portal/assets/8977786/cb32f662-8c3d-4194-bcb0-bbd63c6b5115)

**Comment by JMach1** [comment]
This is not supported as of right now, I've added it to our backlog of requirements.
As a workaround for now, you can set the access option to the whole section the Custom widget is in.

**Comment by bendixon-zelis** [comment]
We too would appreciate this feature. We are aware of the section workaround but it doesn't always work with the wishes of the UI team.
